### ISSUE TITLE
Tracker create_list() function keeps retuning a 403

### ISSUE THREAD

**Issue by jmnmv12** [EasyPost/easypost-python#148]
When I try to use the `Tracker.create_list()` operation I keep receiving the following error:

```
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/env/lib/python3.7/site-packages/easypost/__init__.py", line 800, in create_list
    response, api_key = requestor.request("post", url, params)
  File "/env/lib/python3.7/site-packages/easypost/__init__.py", line 208, in request
    response = self.interpret_response(http_body, http_status)
  File "/env/lib/python3.7/site-packages/easypost/__init__.py", line 273, in interpret_response
    self.handle_api_error(http_status, http_body, response)
  File "/env/lib/python3.7/site-packages/easypost/__init__.py", line 337, in handle_api_error
    raise Error(error.get("message", ""), http_status, http_body)
easypost.Error: This api key is no longer active. Please use a different api key or reactivate this key.
```

I have confirmed multiple times that the key is valid and I am able to do other requests with the same key, for example, I can create individual trackers through the following operation `Tracker.create()` with the same API key.

**Comment by Justintime50** [comment]
Hey @jmnmv12 thanks for reporting this! Are you able to provide a little snippet of code and/or a couple tracking numbers so we could try recreating this on our end?

**Comment by jmnmv12** [comment]
Hey, yeah of course you can recreate it by running this snippet: 
```
easypost.api_key = "MY_API_KEY"
tracker_list = [
    {
        "tracking_code": "EZ1000000001",
        "carrier": "USPS"
    }
]
easypost.Tracker.create_list(tracker_list)
```
I used one of your test tracking codes in this case.

**Comment by jmnmv12** [comment]
I tried to do the same request through Postman and there I don't receive this error, but because I don't know what is the expected request body, I end up receiving a `500 Internal Server Error`. Do you know where I should look to understand what type of request you are expecting on this endpoint? I took a look at your documentation but I couldn't find anything about this.

**Comment by Justintime50** [comment]
In Postman, you could use a raw JSON body like the following which works as intended:

```json
curl -X POST https://api.easypost.com/v2/trackers/create_list \
  -u "API_KEY:" \
  -H "Content-Type: application/json" \
  -d '{"trackers":{
    "0":{"tracking_code":"EZ1000000001", "carrier":"USPS"},
    "1":{"tracking_code":"EZ1000000001", "carrier":"USPS"},
    "2":{"tracking_code":"EZ1000000001", "carrier":"USPS"},
    "3":{"tracking_code":"EZ1000000001", "carrier":"USPS"}
  }}'
```

It looks like there may be a bug in this client library (most likely related to how the API key is passed in this function which is why you are met with the error you are).

**Comment by jmnmv12** [comment]
Okay thank you for the help 👌

**Comment by LoganSimonsen** [comment]
@jmnmv12 I would highly recommend most of our users stick to using our publicly documented endpoint for creating individual trackers https://www.easypost.com/docs/api#create-a-tracker.

Even if you have a high volume of trackers to create, this endpoint should be able to handle high request volume. Feel free to hit up support@easypost.com if you have any problems.

**Comment by Justintime50** [comment]
This should have been corrected and will be included in v7.0.0; however, I'd also recommend what @LoganSimonsen suggested and use our normal trackers endpoint as it's more stable than creating trackers in bulk.
